torch/csrc/jit/constants.cpp

```diff
@@ -149,6 +167,16 @@ RegisterOperators reg({
               push(stack, ts);
               return 0;
             };
+          } else if (type->isSubtypeOf(ListType::ofStrings())) {
+            const auto& ss = node->ss(attr::value);
+            auto vals = c10::impl::GenericList(StringType::get());
```

Comment:
I'm probably missing something obvious: we don't seem to be using `vals` in a lambda.

Reply:
it must be implicitly convertable...